hexrd/ui/calibration/polarview.py

```diff
@@ -286,6 +288,12 @@ def apply_snip(self, img):
             self.snip1d_background = run_snip1d(img)
             # Perform the background subtraction
             img -= self.snip1d_background
+
+            if HexrdConfig().polar_apply_erosion:
+                mask = binary_erosion(
+                    ~self.raw_img.mask, structure=disk(2 * snip_width_pixels())
+                )
+                img[~mask] = 0
```

Comment:
We might want to save the erosion mask out when we export the polar plot as well I guess, right @joelvbernier?

Reply:
Here's what my testing resulted in.  Starting at your line 292:
```
            if HexrdConfig().polar_apply_erosion:
                niter = HexrdConfig().polar_snip1d_numiter
                mask = binary_erosion(
                    ~self.raw_img.mask,
                    structure=rectangle(1, 2.1*niter*snip_width_pixels())
                )
                img[~mask] = 0
```

and just import `rectangle` in lieu of `disk`.

Reply:
Looks like the arguments for `rectangle` need to be integers, so I will cast the second argument to an int like so:
```python
int(2.1*niter*snip_width_pixels())
```

Reply:
This has been fixed.